eVSSIM/MONITOR/SSD_MONITOR_PM/form1.ui.h

```diff
@@ -174,7 +174,7 @@ void Form1::onReceive()
 			}
 			else
 			{
-	      			WriteAmpCount += wb;
+	      			WriteAmpCount = wb;
```

Comment:
1. WriteAmpCount is different from write amplification factor.
2. we need to introduce here another variable that tracks write amplification factor and 2 additional variables: one to count physical block writes and the second that counts logical block counts. Then WriteAmpFactor = PhWritescounter/LogicalWritesCounter.